src/nav/synthetics.yml

```diff
@@ -7,6 +7,8 @@ pages:
         path: /docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring
       - title: Types of monitors
         path: /docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors
+      - title: Limits on monitors
```

Comment:
Not really part of this change, but I think it would be less confusing if we changed this to "licensing and usage" or similar instead of "limits on monitors"